### ISSUE TITLE
Cross domain interactions

### ISSUE THREAD

**Issue by ntromas** [zdk123/SpiecEasi#137]
Hi,

Thanks for the tutorial!
I would like to determine the co-occurrence between two different composition from the same samples (I used two different amplicon sequencing approach from the same samples).
I wonder if, in that case, it is possible to extract the correlation or covariance matrix with ASV names. I am not sure about to what correspond the order as I have two phyloseq element and I am not sure how to extract properly this matrix. I tried as.matrix(symBeta(getOptBeta(result_spieceasi).

Thanks for your help!

Cheers,

Nico

**Comment by zdk123** [comment]
Yes, a labelled output network is requested a lot so I'll add it in the next release. However the output OTU order is the same as the input order. For multi-"domain" it should just be the first dataset concatenated with the second dataset.

**Comment by ntromas** [comment]
Ok thanks!
